startupscript/butane/gcp/metadata-utils.sh

```diff
@@ -3,29 +3,29 @@
 # metadata-utils.sh defines functions for gcp instance attributes and guest attributes. This script is intended to be sourced from other scripts
 # to retrieve or modify instance attributes. It is run on the VM host.
 
-# Retrieves an instance attributes set on the VM.
+# Retrieves an instance attributes set on the VM. If the attribute is not set, it returns the default value.
 function get_metadata_value() {
-  if [[ -z "$1" ]]; then
-    echo "usage: get_metadata_value <tag>"
+  if [[ $# -lt 2 ]]; then
```

Comment:
FWIW, I'm not opposed to the default value being optional (with "" being the default default).
But let's see how we like requiring the caller to specify the default.